File: Python/PRAK603-2210817310007-HafizhPratamaBudiman.py
n1,n2=map(int, input().split())
if (n1!=n2) :
    print('Jumlah tidak sama')
else :
    line1=list(map(int, input().split()))
    line2=list(map(int, input().split()))
for x in range (n1) :
    print(line1[x]*line2[x], end=' ')

File: C/PRAK603-2210817310007-HafizhPratamaBudiman.c
#include <stdio.h>

int main() {

    int b1,b2,n,isi,h;
    scanf("%d %d", &b1, &b2);
    int a1[b1],a2[b2];
    if (b1==b2) {
        for (n=0;n<b1;n++) {
            scanf("%d", &isi);
            a1[n]=isi;
        }
        for (n=0;n<b2;n++) {
            scanf("%d", &isi);
            a2[n]=isi;
        }
        for (n=0;n<b1;n++) {
            h=a1[n]*a2[n];
            printf("%d ", h);
        }
    }
    else {
        printf("Jumlah Tidak Sama");
    }
}